application/client.core/src/app/environment/services/standalone/service.popups.ts

```diff
@@ -72,6 +74,17 @@ export class PopupsService {
         this._opened.delete(guid);
     }
 
+    private _onMouseUp(event: MouseEvent) {
+        if (event.button !== 0) {
+            return;
+        }
+        if (this._opened.size === 0) {
+            return;
+        }
+        this.remove(this._opened.keys().next().value);
```

Comment:
I think it will close any dialog without action. Dialog should be closed only on click on background, but not on dialogs as itself.